### PR TITLE
Adjust  <TokenColorCard/> component to resolve the issue associated with token context

### PR DESCRIPTION
**Pull request by carin91**
Several changes:
1) Changes into the package.json due to the installation of localforage,
2) DB mock with post, set and get using localforage.
3) New tokens structure using the tokens names for the blocks text, surface and strokes.
4) New calculator to find the current token structure from the new tokens structure.
5) The token context includes a new object called TokenWithReference.
6) Several adjustments in tokenColorCard, FieldSetColorCard due to the new structure.
7) All of the forms for the blocks outlets was updated incorporing this new tokens structure.

issues set698 y set700 were addressed in this PR.

### CODE REVIEW

**Review by LinconOssa — APPROVED** [review]
The card is passed but with pending issues that would be (1) When the page is refreshed it is not able to search for the localForAge information, (2) The texts inside the tokens take the information from the theme making the text color change according to the token that you miss (3) what costovunal, corbanca and fondecom are not working in the part of people and colors (palette, texts, surfaces, lines) (4) the storybooks are damaged

File: src/context/TokenContext/index.tsx [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
   IHandleSubmitProps,
   ITokenContextProps,
   TokenActions,
   actionTypes,
 } from "./types";
-import { fetchTokenData } from "@src/mocks/themeService/themeService.mock";
+import {
+  getTokens,
+  updateIdTokens,
+} from "@src/mocks/themeService/themeService.mock";
```
Comment (cmarin001) [inline src/context/TokenContext/index.tsx:19]: We should refrain from using @src, as we have configured aliases for those routes in the tsconfig.json file
image

File: src/context/TokenContext/index.tsx [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+import { tokenCalculator } from "@src/utilities/tokenCalculator";
+import { AppContext } from "@src/context/AppContext";
```
Comment (cmarin001) [inline src/context/TokenContext/index.tsx:21]: We should refrain from using @src, as we have configured aliases for those routes in the tsconfig.json file
image

File: src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { inube } from "@inube/design-system";
 import { paletteMessagesConfig } from "../../config/palette.config";
 import { RenderContentFormPaletteUI } from "./interface";
 import { TokenContext } from "@context/TokenContext";
+import { tokenCalculator } from "@src/utilities/tokenCalculator";
```
Comment (cmarin001) [inline src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx:7]: We should refrain from using @src, as we have configured aliases for those routes in the tsconfig.json file
image

File: src/pages/people/outlets/color/strokes/form/RenderStrokesContentForm/index.tsx [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 import { Appearance } from "@components/cards/FieldsetColorCard/types";
-import { getTokenColor } from "@components/cards/TokenColorCard/styles";
 import { IPeopleMessage } from "@pages/people/outlets/types/people.types";
 import { RenderStrokesContentFormUI } from "./interface";
 import {
   strokesMessagesConfig,
   strokesFormsConfig,
 } from "../../config/Strokes.config";
 import { TokenContext } from "@context/TokenContext";
+import { tokenCalculator } from "@src/utilities/tokenCalculator";
```
Comment (cmarin001) [inline src/pages/people/outlets/color/strokes/form/RenderStrokesContentForm/index.tsx:10]: We should refrain from using @src, as we have configured aliases for those routes in the tsconfig.json file
image

File: src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 import { RenderStrokesWithLinkFormUI } from "./interface";
 import { Appearance } from "@components/cards/FieldsetColorCard/types";
-import { getTokenColor } from "@components/cards/TokenColorCard/styles";
 import { IPeopleMessage } from "@pages/people/outlets/types/people.types";
 import {
   strokesMessagesConfig,
   strokesFormsConfig,
 } from "../../config/Strokes.config";
 import { TokenContext } from "@context/TokenContext";
+import { tokenCalculator } from "@src/utilities/tokenCalculator";
```
Comment (cmarin001) [inline src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx:10]: We should refrain from using @src, as we have configured aliases for those routes in the tsconfig.json file
image

File: src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 import { LoadingAppUI } from "@src/pages/login/outlets/LoadingApp/interface";
```
Comment (cmarin001) [inline src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx:11]: We should refrain from using @src, as we have configured aliases for those routes in the tsconfig.json file
image

File: src/components/cards/TokenColorCard/index.tsx [hunk 7]
```diff
@@ -21,6 +20,7 @@ import { ThemeContext } from "styled-components";
 import { Popup } from "@src/components/feedback/Popup";
 import { MdOutlineEdit } from "react-icons/md";
 import { RenderCategoryGrid } from "@src/components/layout/RenderCategoryGrid";
```
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/cards/TokenColorCard/index.tsx:22]: solved

File: src/components/cards/TokenColorCard/index.tsx [hunk 8]
```diff
@@ -21,6 +20,7 @@ import { ThemeContext } from "styled-components";
 import { Popup } from "@src/components/feedback/Popup";
```
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/cards/TokenColorCard/index.tsx:20]: solved

File: src/components/cards/TokenColorCard/index.tsx [hunk 9]
```diff
@@ -21,6 +20,7 @@ import { ThemeContext } from "styled-components";
...
 import { MdOutlineEdit } from "react-icons/md";
 import { RenderCategoryGrid } from "@src/components/layout/RenderCategoryGrid";
+import { getTokenColor } from "@src/utilities/getTokenColor";
```
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/cards/TokenColorCard/index.tsx:23]: solved

File: src/components/cards/TokenColorCard/styles.ts [hunk 10]
```diff
@@ -1,5 +1,6 @@
 import styled from "styled-components";
 import { inube } from "@inube/design-system";
+import { getTokenColor } from "@src/utilities/getTokenColor";
```
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/cards/TokenColorCard/styles.ts:3]: solved

File: src/components/layout/RenderCategoryGrid/styles.ts [hunk 11]
```diff
@@ -1,5 +1,6 @@
 import styled from "styled-components";
 import { inube } from "@inube/design-system";
+import { getTokenColor } from "@src/utilities/getTokenColor";
```
[the remark this answers is not in the document]
Comment (carin91) [reply src/components/layout/RenderCategoryGrid/styles.ts:3]: solved

File: src/context/TokenContext/index.tsx (hunk 2 above)
Comment (carin91) [reply src/context/TokenContext/index.tsx:21]: solved

File: src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx (hunk 3 above)
Comment (carin91) [reply src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx:7]: solved

File: src/pages/people/outlets/color/strokes/form/RenderStrokesContentForm/index.tsx (hunk 4 above)
Comment (carin91) [reply src/pages/people/outlets/color/strokes/form/RenderStrokesContentForm/index.tsx:10]: solved

File: src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx (hunk 5 above)
Comment (carin91) [reply src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx:10]: solved

File: src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx (hunk 6 above)
Comment (carin91) [reply src/pages/people/outlets/color/strokes/form/RenderStrokesWithLinkForm/index.tsx:11]: solved

**Pull request merged**